ddr/lib/ddr-blobgen/java/genBinaryBlob.cpp

```diff
@@ -804,11 +809,16 @@ BlobFieldVisitor::visitType(Type *type) const
 	size_t bitWidth = 0;
 
 	if (Type::isStandardType(typeName.c_str(), (size_t)typeName.length(), &isSigned, &bitWidth)) {
+#if defined(J9ZOS390) && !defined(OMR_EBCDIC) && defined(__open_xl__)
+		string newType = std::string(isSigned ? "I" : "U") + e2a_string(std::to_string(bitWidth).c_str());
```

Comment:
This has the leak I pointed out in https://github.com/eclipse-omr/omr/pull/7880#discussion_r2515487608.